lib/smack/SmackRep.cpp

```diff
@@ -239,16 +340,24 @@ const Expr* SmackRep::lit(const llvm::Value* v) {
       Expr::lit(exponentPart));
 
   } else if (llvm::isa<llvm::ConstantPointerNull>(v))
-    return Expr::lit(0, width);
+    return Expr::lit(0, MAXSIZE);
 
   else
     return expr(v);
   // assert( false && "value type not supported" );
 }
 
-const Expr* SmackRep::lit(unsigned v) {
+const Expr* SmackRep::lit(int v) {
```

Comment:
This method should be removed, if possible, since it is not possible to determine the right width from a `int` alone.